cnf-certification-test/lifecycle/podsets/podsets.go

```diff
@@ -51,6 +52,27 @@ var WaitForDeploymentSetReady = func(ns, name string, timeout time.Duration) boo
 	return false
 }
 
+var WaitForScaleObjectcaReady = func(ns, name string, timeout time.Duration, groupResourceSchema schema.GroupResource) bool {
+	logrus.Trace("check if sclae object for crs ", ns, ":", name, " is ready ")
+	clients := clientsholder.GetClientsHolder()
+	start := time.Now()
+	for time.Since(start) < timeout {
+		crScale, err := provider.GetUpdatedCrObject(clients.ScalesGetter, ns, name, groupResourceSchema)
+		if err != nil {
+			logrus.Errorf("error while getting the scaling fileds %e", err)
+		} else if !crScale.IsScaleObjectReady() {
+			logrus.Errorf("%s is not ready yet", crScale.ToString())
+		} else {
+			logrus.Tracef("%s is ready!", crScale.ToString())
+			return true
+		}
+
+		time.Sleep(time.Second)
+	}
+	logrus.Error("cr ", ns, ":", name, " is not ready ")
```

Comment:
logrus.Error("timeout waiting for cr ", ns, ":", name, " scaling to be complete")